scripts/west_commands/packages.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2024 Basalte bv
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import subprocess
+import sys
+import textwrap
+from itertools import chain
+from pathlib import Path
+
+from west.commands import WestCommand
+from zephyr_ext_common import ZEPHYR_BASE
+
+sys.path.append(os.fspath(Path(__file__).parent.parent))
+import zephyr_module
+
+
+def in_venv() -> bool:
+    return sys.prefix != sys.base_prefix
+
+
+class Packages(WestCommand):
+    def __init__(self):
+        super().__init__(
+            "packages",
+            "manage packages for Zephyr",
+            "List and Install packages for Zephyr and modules",
+            accepts_unknown_args=True,
+        )
+
+    def do_add_parser(self, parser_adder):
+        parser = parser_adder.add_parser(
+            self.name,
+            help=self.help,
+            description=self.description,
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            epilog=textwrap.dedent(
+                """
+            Listing packages:
+
+                Run 'west packages <manager>' to list packages for Zephyr and
+                modules for a given package manager.
```

Comment:
```
                Run 'west packages <manager>' to list all dependencies
                available from a given package manager, already
                installed and not. These can be filtered by module, see...
```

Again, "packages" feels too generic in this sentence. No need to repeat it when it's already the command name, just use the more specific "dependencies" or "requirements" terms already in use.


Ambiguous parsing:

> to list (packages for Zephyr) and modules for a given package manager.

versus

> to list packages for (Zephyr and modules) for a given package manager.

Less likely because you have to bend "modules" a bit but you could even read it as:

> to list packages for Zephyr and (modules for a given package manager).

=> sentence too long